konduit-serving-rpm/src/main/resources/etc/profile.d/konduit-serving-env.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# This file defines the environment variables needed to run the konduit serving admin command.
+
+export KONDUIT_SERVING_BASE_ARTIFACT_NAME=konduit-serving
+export KONDUIT_SERVING_DIRECTORY_STRUCTURE=konduit/serving
+export KONDUIT_SERVING_VERSION=0.1.0-SNAPSHOT
+export KONDUIT_SERVING_BACKEND=cpu
```

Comment:
We don't have a concept of a backend anymore. Maybe make sure we clean up old assumptions while we're at it here.

Reply:
This isn't an old assumption, these settings are for when we want to use gpu based python libraries or cpu based ones. This is set while we're building with `-Dchip=cpu` or `-Dchip=gpu`